docs/man/man1/kubectl-drain.1

```diff
@@ -33,6 +33,10 @@ will make the node schedulable again.
 
 .SH OPTIONS
 .PP
+\fB\-\-delete\-local\-data\fP=false
+    Continue even if there are pods with local data (EmptyDir).
```

Comment:
Note: I think the flag name is fine, I'm just referring to the way you described it (here and elsewhere in the PR).